docs/docs/installation/cache.mdx

```diff
@@ -53,6 +53,13 @@ FEATURE_FLAGS = {
 }
 ```
 
+By default thumbnails are rendered using the `THUMBNAIL_SELENIUM_USER` user account. To render thumbnails as the
+logged in user (e.g. in environments that are using user impersonation), use the following configuration:
+
+```python
+THUMBNAIL_EXECUTE_AS = [ExecutorType.INITIATOR]
```

Comment:
nit: at first I struggled with the name `INITIATOR`, after reading the entire PR I got more used to it :)
still do you think `ExecutorType.CURRENT_USER` or similar could be more explicit?

Reply:
I originally had `CURRENT_USER`, but for some reason I decided `INITIATOR` was better. But now that you said it, yeah, `CURRENT_USER` makes much more sense. I'll update it.